### DISCUSSION TITLE
Is a Stripe Connect account required for a standalone non-profit in Houdini 2?

### DISCUSSION THREAD

**Discussion by BenSturmfels** [houdiniproject/houdini#1062] [Q&A] [answered]
Hi Folks,

I was just testing out the `commitchange/houdini` codebase and I see that a Stripe Connect account is required for routing payments between multiple parties as CommitChange would do in the hosted service. Is Stripe Connect also a requirement in the Houdini 2 codebase or is regular Stripe account ok? I suspect it is, but just wanted to be sure I wasn't missing something.

Regard,
Ben

**Comment by wwahammy** [comment, marked as answer]
@BenSturmfels A regular account should be okay, at least that's the intent.

**Reply by BenSturmfels** [reply]
Thanks!
